WordPress-VIP-Go/ruleset.xml

```diff
@@ -230,12 +216,12 @@
 	<rule ref="WordPress.WP.EnqueuedResources.NonEnqueuedScript">
 		<type>warning</type>
 		<severity>3</severity>
-		<message>Scripts must be registered/enqueued via `wp_enqueue_script`. This can improve the site's performance due to script concatenation</message>
+		<message>Scripts must be registered/enqueued via `wp_enqueue_script`. This can improve the site's performance due to script concatenation.</message>
```

Comment:
Since it's a Warning, and currently only severity 3, should we change `must` to `should`?

Reply:
I think so.